srcpkgs/taizen/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'taizen'
+pkgname=taizen
+version=0.1.0
+revision=1
+_githash=d86da2ce6c687401fe8e6e34fc69cd22aaba72e1
```

Comment:
you use a git commit dated Aug 5, 2018.
`version=0.0.0.20180805`

Reply:
see also https://github.com/voidlinux/void-packages/pull/6707#discussion_r121899848